bin/silius/src/metrics.rs

```diff
@@ -0,0 +1,480 @@
+use async_trait::async_trait;
```

Comment:
Should we move this to new crate - metrics?

Reply:
Do you mean a new workspace crate?

Reply:
Yes. What do you think, does it make sense?

Reply:
Ok. Make sense